ReactiveCocoaTests/Swift/SignalSpec.swift

```diff
@@ -625,6 +625,40 @@ class SignalSpec: QuickSpec {
 				observer.sendNext("d")
 				expect(values) == [ "a", "cc", "d" ]
 			}
+
+			it("should not store strong reference to previously passed items") {
+				var disposedItems: [Bool] = []
+
+				struct Item {
```

Comment:
It doesn't make sense to talk about "strong references" with a struct, which has value semantics. I would change this to a `class`.